fastlane/Fastfile

```diff
@@ -42,7 +42,7 @@ import "./ScreenshotFastfile"
 ########################################################################
 Dotenv.load('~/.wpandroid-env.default')
 ENV[GHHELPER_REPO="wordpress-mobile/WordPress-Android"]
-ENV["PROJECT_ROOT_FOLDER"]="./"
+ENV["PROJECT_ROOT_FOLDER"]=File.dirname(File.expand_path(__dir__)) + "/"
```

Comment:
I also wonder if we need to keep the trailing `/`.

If we were 100% sure all of the path management is done with Ruby APIs instead of string interpolation we wouldn't, but I'm not sure of that, so maybe it's best to err on the side of caution?

Reply:
Yeah, I think so. It's used in many different places and I'm not confident about changing it right now. It would be great if @AliSoftware manages to tackle this while working on the release-toolkit and Fastfile refactoring/alignment project.

Reply:
Yeah there's an issue logged about this already – https://github.com/wordpress-mobile/release-toolkit/issues/205 – and it would be nice to tackle it at some point, but currently felt low priority. So I agree we should keep the trailing slash (and the env var) so far until we get time to tackle this.

Reply:
👍